templates/security/cve/_cve-card.html

```diff
@@ -1,80 +1,91 @@
+          </span>
+        </div>
+      </div>
+      <div class="col-3 col-medium-2">
+        {% if cve.summarized_status %}
+          {% set status = cve.summarized_status %}
+          {% if status.name == "Some fixed" %}
+            <p>
+              Some fixes available <span class="u-text--muted">{{ status.fixed_count }} of {{ status.total_count }}</span>
+            </p>
+          {% elif status.name == "Needs evaluation" %}
+            <p>
+              <i class="p-icon--help"></i> <span>Needs evaluation</span>
+            </p>
+          {% elif status.name == "Fixed" %}
+            <p>
+              <i class="p-icon--success"></i> <span>Fixed</span>
+            </p>
+          {% elif status.name == "Vulnerable" %}
+            <p>
+              <i class="p-icon--warning"></i> <span>Vulnerable</span>
+            </p>
+          {% elif status.name == "DNE" %}
+            <p>
+              <span>Not in release</span>
+            </p>
           {% else %}
-          <i class="p-icon--placeholder"></i>
+            {{ status }}
           {% endif %}
-          <span>{{ cve.priority.split() | first | capitalize }} priority</span>
-        </span>
+        {% endif %}
       </div>
     </div>
-    <div class="col-3 col-medium-2">
-      {% set status = cve.summarized_status %}
-      {% if status.name == "Some fixed" %}
-        <p>Some fixes available <span class="u-text--muted">{{ status.fixed_count }} of {{ status.total_count }}</span></p>
-      {% elif status.name == "Needs evaluation" %}
-        <p>
-          <i class="p-icon--help"></i> <span>Needs evaluation</span>
-        </p>
-      {% elif status.name == "Fixed" %}
-        <p>
-          <i class="p-icon--success"></i> <span>Fixed</span>
-        </p>
-      {% elif status.name == "Vulnerable" %}
-        <p>
-          <i class="p-icon--warning"></i> <span>Vulnerable</span>
-        </p>
-      {% elif status.name == "DNE" %}
-        <p>
-          <span>Not in release</span>
-        </p>
-      {% else %}
-      {{status}}
-      {% endif %}
-    </div>
-  </div>
-  <div class="row p-section--shallow">
-    <div class="col-6 col-medium-4">
-      <p>{{ cve.description|truncate(235) }}</p>
-    </div>
-    <div class="col-3 col-medium-2">
-      <p class="p-text--small-caps u-text--muted u-no-margin--top">{{ cve.packages | length}} affected packages</p>
-      <p class="u-text--muted">
-        {% for package in cve.packages %}
-          {% if cve.packages | length > 6 %}
-            {% if loop.index < 5 %}
-              {{ package.name }}, 
-            {% elif loop.index == 5 %}
-              {{ package.name }}... 
-            {% endif %}
-          {% else %}
-            {% if loop.last %}
-              {{ package.name }}
+    <div class="row p-section--shallow">
+      <div class="col-6 col-medium-4">
+        <p>{{ cve.description|truncate(235) }}</p>
+      </div>
+      <div class="col-3 col-medium-2">
```

Comment:
Looking at the mock-up I believe this should be hidden when the detailed view is toggled 
![image](https://github.com/canonical/ubuntu.com/assets/17607612/5e2d80fb-0336-430d-865c-93bffbb5baa9)